src/utils/fireQuery.ts

```diff
@@ -474,7 +480,7 @@ export const getDatalogQueryComponents = ({
         };
       },
       pull: `(pull ?${returnNode} [:block/string :node/title :block/uid])`,
-      label: "text",
+      label: selections.find(s => s.text === "node")?.label || "text",
```

Comment:
It's a little odd for default values to be based on the input, but I think this is probably the cleanest way to handle this! I think we should send it 🔥

Reply:
An improvement on this, to make extensible with a configurable `returnNode` (like in Query Drawer), is to do `find(s => s.text === returnNode)`

The second part on that is for somehow `ALIAS_TEST` to take the returnNode into account - this could all be revisited when deciding whether query blocks should have a configurable return node or not